src/main/java/com/actiontech/dble/services/BackendService.java

```diff
@@ -166,10 +163,20 @@ private void handleTaskQueue(ThreadContext threadContext) {
             try {
                 // handleData
                 while ((task = taskQueue.poll()) != null) {
-                    DbleThreadPoolProvider.beginProcessBackendBusinessTask();
-                    consumeReadingData(task);
-                    this.consumeSingleTask(task);
-                    ThreadPoolStatistic.getBackendBusiness().getCompletedTaskCount().increment();
+                    final Service service = task.getService();
+                    try {
+                        if (threadName.contains(DbleServer.BACKEND_WORKER_NAME)) {
```

Comment:
hardcode isn't suggested. "put" method can return bool value to prevent idempotent and hardcode.

Reply:
This piece of code records only the threads of BACKEND_WORKER